Maple2.Server.Game/Model/Field/Actor/FieldActor.cs

```diff
@@ -21,12 +24,15 @@ internal sealed class FieldActor : IActor {
     public Transform Transform { get; init; }
     public BuffManager Buffs { get; }
     public Stats Stats { get; }
+    public AnimationState AnimationState { get; init; }
 
-    public FieldActor(FieldManager field) {
+    public FieldActor(FieldManager field, NpcMetadataStorage npcMetadata) {
         Field = field;
         Stats = new Stats(0, 0);
         Buffs = new BuffManager(this);
         Transform = new Transform();
+        NpcMetadata = npcMetadata;
+        AnimationState = new AnimationState(this, "");
```

Comment:
Is this intentionally meant to be empty model name on construct?

Reply:
Yeah, as far as I can tell it's an actor exclusively used for caster-less region skills & effects, so we don't need to care about if it has a valid model or not. Setting it to empty string just makes the update function return early so it doesn't do anything unnecessary.

Reply:
let's make that parameter have a default value of string.Empty then rather than passing an empty string